docs/docsite/rst/playbook_guide/playbook_pathing.rst

```diff
@@ -21,11 +21,13 @@ Task paths
 
 Relative paths used in a task typically refer to remote files and directories on the managed nodes. However, paths passed to lookup plugins and some paths used in action plugins such as the "src" path for the :ansplugin:`template <ansible.builtin.template#module>` and :ansplugin:`copy <ansible.builtin.copy#module>` modules refer to local files and directories on the control node.
 
-Resolving local relative paths
-------------------------------
+
+Resolving local lookup and action plugin relative paths
```

Comment:
Maybe it's because English isn't my native language, but I find the 'for' a bit confusing, as if this is about the paths of the plugins, and not about how such plugins resolve local paths.

How about replacing 'for' by 'in', or rearranging the sentence to something like 'How lookup and action plugins resolve local relative paths'?